tests/e2e/test_rosacli_cluster.go

```diff
@@ -332,6 +332,97 @@ var _ = Describe("Edit cluster",
 					To(ContainSubstring(
 						"ERR: Schedule '\"5 5\"' is not a valid cron expression"))
 			})
+
+		It("can via rosa-cli - [id:60275]",
+			labels.Critical, labels.Runtime.Day2,
+			func() {
+				By("Check the cluster is private cluster")
+				private, err := clusterService.IsPrivateCluster(clusterID)
+				Expect(err).To(BeNil())
+				if !private {
+					SkipTestOnFeature("private")
+				}
+				isSTS, err := clusterService.IsSTSCluster(clusterID)
+				Expect(err).To(BeNil())
+				isHostedCP, err := clusterService.IsHostedCPCluster(clusterID)
+				Expect(err).To(BeNil())
+
+				defer func() {
```

Comment:
Put this defer after the edit command. Otherwise it will fail if the cluster is a private link cluster